### PR TITLE
Proofread README.md

### PR DESCRIPTION
**Pull request by abesto**
I was reading the readme while waiting for the modlist to install, and tripped on a few phrases. Figured may as well contribute a PR while I wait for the downloads.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Just do it. This entire Modlist is in English and 99% of all mods you will find
 
 Open the Steam Properties window, navigate to the _Language_ tab and select _English_ from the dropdown menu.
 
-#### Clean Skyrim <Special Edition>
```
Comment (abesto) [inline README.md:108]: `<stuff>` is treated as an HTML tag, so the "Special Edition" text never showed up before. I guess we want it?

File: README.md [hunk 2]
```diff
@@ -114,19 +114,19 @@ I highly recommend uninstalling the game through Steam, deleting the game folder
 After you have done everything above and got a clean SSE installation ready, start the Launcher and and let it do the initial graphics check. Do not worry about this part as the installation will replace this graphics settings.
 Start the game and exit once you're in the main menu.
 
-### Downgrade Skyrim Anniversary Edition to Special Edition
+#### Downgrade Skyrim Anniversary Edition to Special Edition
```
Comment (abesto) [inline README.md:117]: The heading level here and the table of contents disagree on what level this heading should be. I figure this should be under "Installation".

File: README.md [hunk 3]
```diff
@@ -114,19 +114,19 @@ I highly recommend uninstalling the game through Steam, deleting the game folder
...
-The Anniversary Edition and Anniversary Update make a number of fundamental changes which mean many critical tools, such as SKSE and mods that rely on those tools, need to be updated.
+The Anniversary Edition and Anniversary Update makes a number of fundamental changes which means many critical tools, such as SKSE and mods that rely on those tools, need to be updated.
 
 If your Skyrim Special Edition received an update through Steam anytime after November 11, 2021, you have the Anniversary Update. Note that it might be automatically updated by Steam even if you have disabled updates.
 
 In order to install Serenity 2, you must use the [FULL AE downgrade patcher](https://www.nexusmods.com/skyrimspecialedition/mods/57618) (**NOT** 'Best of Both Worlds')
 
 After Serenity 2 installation would be completed, you may update your Steam Skyrim to AE if you like - Serenity would continue to use own copy of SSE files.
 
-Please do not ask when any AE or the AE update will be supported.
```
Comment (abesto) [inline README.md:129]: I *think* this keeps the intended meaning?

File: README.md [hunk 4]
```diff
@@ -191,9 +193,9 @@ A few other ENB suggestions are:
 
 ### Requiem
 
-[Requiem](https://www.nexusmods.com/skyrim/mods/19281) is a project with the goal of turning Skyrim into a better roleplaying experience by making game more immersive, semi-realistic and coherent, with the tend to find a compromiss between old school mechanics and more modern approaches.
+[Requiem](https://www.nexusmods.com/skyrim/mods/19281) is a project with the goal of turning Skyrim into a better roleplaying experience by making game more immersive, semi-realistic and coherent, and tends to find a compromise between old school mechanics and more modern approaches.
```
Comment (abesto) [inline README.md:196]: I had to guess with "tends" here; the original meaning was not at all clear to me.

File: README.md (hunk 1 above)
Comment (azraerl) [reply README.md:108]: Agree

**Review by azraerl — APPROVED** [review]
Looks good, thank you for contribution 👍🏽

**Pull request merged**